joomla/joomla-cms
# Fix the multiple instances bug for tinyMCE

Pull Request for Issue reported https://github.com/joomla/joomla-cms/pull/12561#issuecomment-281817220 by @svenbluege .

### Summary of Changes
Multiple instances of tinyMCE fail to initialise correctly.


### Testing Instructions

Make sure that the default editor is tinyMCE

Edit `administrator/components/com_content/models/forms/article.xml` and add after 

``` xml
    <fieldset addfieldpath="/administrator/components/com_categories/models/fields" >
```

this:

``` xml
        <field
                name="test2"
                label ="Test Field"
                type="editor"
                width="300"
                filter="safehtml"
                editor="tinymce"
                buttons="true"/>
        <field
                name="test3"
                label ="Test Field"
                type="editor"
                width="300"
                filter="safehtml"
                editor="tinymce"
                buttons="true"/>
```

Edit `administrator/components/com_content/views/article/tmpl/edit.php` and add after

``` html
                <fieldset class="adminform">
                    <?php echo $this->form->getInput('articletext'); ?>
                </fieldset>
```

this:

``` html
                <fieldset class="adminform">
                    <?php echo $this->form->getInput('test2'); ?>
                </fieldset>
                <fieldset class="adminform">
                    <?php echo $this->form->getInput('test3'); ?>
                </fieldset>
```

### Expected result



### Actual result



### Documentation Changes Required


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/joomla/joomla-cms/14194)
